pkg/kubelet/metrics/metrics.go

```diff
@@ -488,6 +493,26 @@ var (
 		},
 		[]string{"container_type", "code"},
 	)
+	// StartedHostProcessContainersTotal is a counter that tracks the number of hostprocess container creation operations
+	StartedHostProcessContainersTotal = metrics.NewCounterVec(
+		&metrics.CounterOpts{
+			Subsystem:      KubeletSubsystem,
+			Name:           StartedHostProcessContainersTotalKey,
+			Help:           "Cumulative number of hostprocess containers started",
```

Comment:
Since these metrics are windows-only, can we include that in the HELP text?

Reply:
I added `This metric will only be collected on Windows and requires WindowsHostProcessContainers feature gate to be enabled.` to the help text for both metrics.